app/controllers/graders_controller.rb

```diff
@@ -87,6 +87,44 @@ def global_actions
       end
     end
 
+    # '%w' syntax is a Ruby idiom that, preferred over ["...", "...", ...] syntax for its conciseness and readability
```

Comment:
You should remove all comments here. These are good for learning purposes but not in the final code.